src/physics/mod_dust.t

```diff
@@ -48,6 +48,9 @@ module mod_dust
   !> This can be turned off for testing purposes
   logical :: dust_backreaction = .true.
 
+  !> Add source terms from gravity field to dust's equation of motion
+  logical :: dust_gravity = .false.
```

Comment:
What about controllong gravity through the hd_gravity flag? If you want to have the option to turn it on/off for dust only you can of course keep it.

Reply:
On second thoughts I think it might create more problems to silently introduce this additional parameter than to force consistent physics and using only `hd_gravity`. I'm going with this.